parcels/tools/converters.py

```diff
@@ -53,9 +53,19 @@ def reltime(self, time):
             return (time - self.time_origin) / np.timedelta64(1, 's')
         elif self.calendar in _get_cftime_calendars():
             if isinstance(time, (list, np.ndarray)):
-                return np.array([(t - self.time_origin).total_seconds() for t in time])
+                try:
+                    return np.array([(t - self.time_origin).total_seconds() for t in time])
+                except np.core._exceptions.UFuncTypeError:
+                    raise ValueError(f"Cannot subtract 'time' (a {type(time)} object) from "
+                                     f"a {self.calendar} calendar.\n"
+                                     f"Provide 'time' as a {type(self.time_origin)} object?")
             else:
-                return (time - self.time_origin).total_seconds()
+                try:
+                    return (time - self.time_origin).total_seconds()
+                except ValueError:
+                    raise ValueError(f"Cannot subtract 'time' (a {type(time)} object) from "
+                                     f"a {self.calendar} calendar.\n"
+                                     f"Provide 'time' as a {type(self.time_origin)} object?")
```

Comment:
this string-concatenate will generate another syntax-error - isn't there a '\' missing at the end of the line ?

Reply:
All lines have a beginning and ending `"`, or am I missing something?

Reply:
well, so I just tried 30 min to get an up-to-date version of flake8, but I am stuck with the 1.6.0 version. This seems to be an error interpretation coming with a recent flake8 version. Because parcels relies on flake8 to finish, the linter error needs to be resolved.

Now, I never use this `f"..."` notation, because I am used to the `"... %s ..." % (string)` notation or the `"... {} ... {} ...".format(...)` notation because my world is C/C++. So, if it was me, I would just rewrite the whole thing.

If you like this new notation, then that's also totally fine, but then I can't help you resolve the linter error here - again, my resolution would be a rewrite not using that notation in the first place, and not do line-breaks on those error message. Sorry for that.